lib/features/dataset/display.dart

```diff
@@ -34,9 +35,12 @@ import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:rattle/constants/app.dart';
 import 'package:rattle/constants/markdown.dart';
 import 'package:rattle/constants/spacing.dart';
+import 'package:rattle/features/dataset/config.dart';
```

Comment:
This should not be required for a single constant - the constant belongs in constants/spacing.dart - use one from there already

Reply:
done